src/nxt_conf.c

```diff
@@ -459,7 +463,7 @@ nxt_conf_type(nxt_conf_value_t *value)
 
 
 nxt_conf_value_t *
-nxt_conf_get_path(nxt_conf_value_t *value, nxt_str_t *path)
+nxt_conf_get_path(const nxt_conf_value_t *value, const nxt_str_t *path)
```

Comment:
If we touch it, I feel it's like this:
```
const nxt_conf_value_t *
nxt_conf_get_path(const nxt_conf_value_t *value, const nxt_str_t *path)
```
Since both the value and the return value are read-only.

Reply:
Ah, let me check...

Reply:
OK, so that's going to be a much larger intrusive change. You have to start making function variables `const` then you need to start making various structure members `const`, or you introduce a tonne of casting...

Either way this would definitely be a separate commit.

Reply:
Yep, it will introduce much change. Or stay them the same.

Reply:
Btw, others like just adding `const` look good. They can be released first.

Reply:
I can certainly drop _this_ change for now, it doesn't prevent my initial goal with these patches...

Reply:
Dropped.